src/libraries/System.Net.Requests/tests/System.Net.Requests.Tests.csproj

```diff
@@ -4,9 +4,12 @@
     <IncludeRemoteExecutor>true</IncludeRemoteExecutor>
     <TargetFrameworks>$(NetCoreAppCurrent)</TargetFrameworks>
     <DefineConstants>$(DefineConstants);NETSTANDARD</DefineConstants>
+    <IgnoreForCI Condition="'$(TargetOS)' == 'Browser'">true</IgnoreForCI>
   </PropertyGroup>
   <ItemGroup>
     <Compile Include="AssemblyInfo.cs" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetsBrowser)' != 'true'">
```

Comment:
this won't work since `TargetsBrowser` is only set when `TargetFrameworks` includes  `$(NetCoreAppCurrent)-Browser` which is not the case here. It doesn't have an effect since we never run this assembly due to the `IgnoreForCI` but I would remove these additional checks, we didn't add them for other PNSE cases like System.Net.Ping.Functional.Tests.csproj either.

Reply:
Updated